src/cli/test_command.zig

```diff
@@ -718,6 +739,70 @@ pub const TestCommand = struct {
             // Treat arguments as filters and scan the codebase
             const filter_names = if (ctx.positionals.len == 0) &[0][]const u8{} else ctx.positionals[1..];
 
+            if (ctx.test_options.include) |include_glob_pattern| {
+                const glob = @import("../glob.zig");
```

Comment:
can you use `bun.Glob` or something which is not an import path? if we move files around later, this will be something that needs to be updated